src/sentry/snuba/events.py

```diff
@@ -186,6 +188,7 @@ class Columns(Enum):
         event_name="transaction",
         transaction_name="transaction_name",
         discover_name="transaction",
+        issue_platform_name="transaction_name",
```

Comment:
This gets already gets mapped to `tags['transaction']` in snuba - just adding it here explicityl.